jsonschema/current/Interfaces.schema.json

```diff
@@ -17,13 +17,9 @@
                 {"$ref": "dungeoneer/entities/Sprite.schema.json"},
                 {"$ref": "dungeoneer/entities/projectiles/Projectile.schema.json"},
                 {"$ref": "dungeoneer/entities/triggers/BasicTrigger.schema.json"},
```

Comment:
While you are at it, what do you think of grouping these `BasicTrigger` related triggers into a `BasicTriggerLike` group?

Reply:
As discussed, not used at the moment. So we can skip it.